TLS feature does not compile using the software AES backend

Vibe report:D, im using a Ryzen 2200G if that matters

Description: When using rand_aes 0.6.0 with features = ["tls", "tls_aes128_ctr128"] on Windows, the crate fails to compile with:

```
error[E0599]: no function or associated item named `zeroed` found for struct `runtime::Aes128Ctr128` in the current scope
   --> src/tls.rs:87:89
    |
87 |     pub(super) static RNG: core::cell::LazyCell<Prng> = core::cell::LazyCell::new(Prng::zeroed);
    |                                                                                         ^^^^^^ function or associated item not found in `runtime::Aes128Ctr128`
```

Environment:

OS: Windows 11
Rust version: 1.95.0-nightly (905b92696 2026-01-31)
rand_aes version: 0.6.0
Features enabled: tls, tls_aes128_ctr128
Minimal reproduction:

```
toml
[dependencies]
rand_aes = { version = "0.6", features = ["tls", "tls_aes128_ctr128"] } (Korangar)
```
Root cause analysis: The tls module uses Prng::zeroed() to initialize the thread-local RNG. On Windows, Aes128Ctr128 uses a software fallback implementation (Aes128Ctr128Inner) that doesn't derive Zeroable, causing the method to be missing.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.6.1] - 2026-03-28
+
+### Fixed
+
+- Fixes the TLS feature when using the software backend
+
 ## [0.6.0] - 2025-10-31
 
 ### Changed

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -8,7 +8,7 @@ license = "Apache-2.0"
 readme = "README.md"
 documentation = "https://docs.rs/rand_aes"
 repository = "https://github.com/hasenbanck/rand_aes"
-version = "0.6.0"
+version = "0.6.1"
 edition = "2024"
 rust-version = "1.91"
 

--- a/src/runtime.rs
+++ b/src/runtime.rs
@@ -141,6 +141,21 @@ enum Aes128Ctr128Inner {
 pub struct Aes128Ctr128(Aes128Ctr128Inner);
 
 impl Aes128Ctr128 {
+    // This function is needed for the TLS.
+    #[cfg(all(feature = "tls", feature = "tls_aes128_ctr128"))]
+    pub(crate) fn zeroed() -> Self {
+        match has_hardware_acceleration() {
+            true => {
+                let hardware = Aes128Ctr128Hardware::zeroed();
+                Self(Aes128Ctr128Inner::Hardware(Box::new(hardware)))
+            }
+            false => {
+                let software = Aes128Ctr128Software::zeroed();
+                Self(Aes128Ctr128Inner::Software(Box::new(software)))
+            }
+        }
+    }
+
     pub(crate) fn jump_impl(&self) -> Self {
         let inner = match &self.0 {
             Aes128Ctr128Inner::Hardware(this) => {
@@ -231,6 +246,21 @@ enum Aes256Ctr64Inner {
 pub struct Aes256Ctr64(Aes256Ctr64Inner);
 
 impl Aes256Ctr64 {
+    // This function is needed for the TLS.
+    #[cfg(all(feature = "tls", feature = "tls_aes256_ctr64"))]
+    pub(crate) fn zeroed() -> Self {
+        match has_hardware_acceleration() {
+            true => {
+                let hardware = Aes256Ctr64Hardware::zeroed();
+                Self(Aes256Ctr64Inner::Hardware(Box::new(hardware)))
+            }
+            false => {
+                let software = Aes256Ctr64Software::zeroed();
+                Self(Aes256Ctr64Inner::Software(Box::new(software)))
+            }
+        }
+    }
+
     pub(crate) fn from_seed_impl(key: [u8; 32], nonce: [u8; 8], counter: [u8; 8]) -> Self {
         match has_hardware_acceleration() {
             true => {
@@ -297,6 +327,21 @@ enum Aes256Ctr128Inner {
 pub struct Aes256Ctr128(Aes256Ctr128Inner);
 
 impl Aes256Ctr128 {
+    // This function is needed for the TLS.
+    #[cfg(all(feature = "tls", feature = "tls_aes256_ctr128"))]
+    pub(crate) fn zeroed() -> Self {
+        match has_hardware_acceleration() {
+            true => {
+                let hardware = Aes256Ctr128Hardware::zeroed();
+                Self(Aes256Ctr128Inner::Hardware(Box::new(hardware)))
+            }
+            false => {
+                let software = Aes256Ctr128Software::zeroed();
+                Self(Aes256Ctr128Inner::Software(Box::new(software)))
+            }
+        }
+    }
+
     pub(crate) fn jump_impl(&self) -> Self {
         let inner = match &self.0 {
             Aes256Ctr128Inner::Hardware(this) => {

